codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -131,11 +154,32 @@ public HttpHeaders clear() {
         }
 
         @Override
+        public Entry<CharSequence, CharSequence> forEachEntry(EntryVisitor<CharSequence> visitor) throws Exception {
+            return null; // Since this is an empty header collection
+        }
+
+        @Override
         public Iterator<Entry<String, String>> iterator() {
             return entries().iterator();
         }
     };
 
+    private final EntryVisitor<CharSequence> setAllVisitor = new EntryVisitor<CharSequence>() {
```

Comment:
Yep I saw that PR & the change makes sense. Will make the change here too. 
Quick question:
Not that it matters in this case, but HttpHeaders isn't threadsafe, correct?